accord-core/src/main/java/accord/local/CommandStore.java

```diff
@@ -91,12 +90,17 @@ public int generation()
         return generation;
     }
 
-    public boolean hashIntersects(RoutingKey key)
+    public boolean hashIntersects(SortableKey key)
     {
         return ShardedRanges.keyIndex(key, numShards) == shardIndex;
     }
 
-    public boolean intersects(Keys keys, KeyRanges ranges)
+    public boolean hashIntersects(Sortable routable)
+    {
+        return routable instanceof Range || hashIntersects((SortableKey) routable);
+    }
+
+    public boolean intersects(Keys keys, Ranges ranges)
```

Comment:
When do we have `Ranges` that are ranges of `Seekables` no `Routables` (and thus work with `Keys`? I checked and this seems unused and I think is also wrong.

Also maybe `AbstractKeys.any` shouldn't compile with this attempted usage?

Reply:
We don't, but `Ranges` of `Routables` are comparable with `Key` - both `Seekable` and `Routable` are `Sortable`. This just asks, do any of the `Key` occur in any of the `RoutingKey` bounds of the `Ranges`

Reply:
My confusion is that a Routable is something describing something over the partitioning schema and a Seekable is describing something over some logical range of values for a key.

When would it ever make sense to mix these in collections or compare them to one another?